Packs/Symantec_Advanced_Threat_Protection/Integrations/integration-Symantec_Advanced_Threat_Protection.yml

```diff
@@ -28,6 +28,45 @@ configuration:
   defaultvalue: ""
   type: 8
   required: false
+- display: Incident data source
+  name: fetch_incidents_type
+  defaultvalue: incidents
+  type: 15
+  required: false
+  options:
+  - incidents
+  - events
+  - incidentevents
+- display: Query string for fetch incidents
+  name: fetch_incidents_query
+  defaultvalue: ""
```

Comment:
is there a default query? can we also refer to the docs in the `additionalinfo` key?